dracut/30ignition/ignition-setup-pre.sh

```diff
@@ -39,4 +39,8 @@ if [ "${oem_cmdline}" = "pxe" ]; then
   oem_cmdline="metal"
 fi
 
+if [ "${oem_cmdline}" = "ionoscloud" ]; then
+  oem_cmdline="qemu"
+fi
```

Comment:
Can be removed when https://github.com/coreos/ignition/pull/1957 is merged and released